src/Parallel/centrality/betweenness.jl

```diff
@@ -2,9 +2,15 @@ betweenness_centrality(g::AbstractGraph, vs=vertices(g), distmx::AbstractMatrix=
 parallel == :distributed ? distr_betweenness_centrality(g, vs, distmx; normalize=normalize, endpoints=endpoints) : 
 threaded_betweenness_centrality(g, vs, distmx; normalize=normalize, endpoints=endpoints)
     
-betweenness_centrality(g::AbstractGraph, k::Integer, distmx::AbstractMatrix=weights(g); normalize=true, endpoints=false, parallel=:distributed) = 
-parallel == :distributed ? distr_betweenness_centrality(g, sample(vertices(g), k), distmx; normalize=normalize, endpoints=endpoints) :
-threaded_betweenness_centrality(g, sample(vertices(g), k), distmx; normalize=normalize, endpoints=endpoints)
+function betweenness_centrality(
+    g::AbstractGraph, k::Integer, distmx::AbstractMatrix=weights(g);
+    normalize=true, endpoints=false, parallel=:distributed,
+    rng::Union{Nothing, AbstractRNG}=nothing, seed::Union{Nothing, Integer}=-1
```

Comment:
```suggestion
    rng::Union{Nothing, AbstractRNG}=nothing, seed::Union{Nothing, Integer}=nothing
```
In the previous place where we used `rng_from_rng_or_seed` we set the default or `seed` to `nothing`, we probably should do that here too.

Reply:
Replaced all `seed=-1` values to `seed=nothing`, and fixed docstrings.